### PR TITLE
script for retrieving transcripts from TedTalks

### PR DESCRIPTION
**Pull request by dag7dev**
This script fetch all json transcript files which are on TedTalks.com.
All the json files are parsed.

The script may take a lot to run, depending on your cpu and the requests time which are unavoidable.

Note: there is a log file. This is mandatory, since nothing will be shown on Terminal in case of error. Every piece of code related to the instruction "trace" shouldn't be removed (comment them instead) as they provide what is going on.

### CODE REVIEW

File: MITADS/ted_importer.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    # managing beautifulsoup res
+    baseurl = "https://www.ted.com"
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36'}
+
+    pagenumber = 1
+    file_counter = 0
+
+    print("TedImporter")
+    print("NOTE: This may take A LOT!")
```
Comment (Mte90) [inline MITADS/ted_importer.py:46]: this can be removed

File: MITADS/ted_importer.py [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+            pagenumber += 1
+        else:
+            the_end = True
+
+    print("Created and parsed " + str(file_counter) + " files.")
+    print("Import from ted.com completed! See " + outfilepath)
+    print("Time elapsed: " + str( int( int( time.time() - start_time ) / 60 ) ) + " minutes" )
+
+    trace("Created and parsed " + str(file_counter) + " files.")
```
Comment (Mte90) [inline MITADS/ted_importer.py:90]: Code duplicated

File: MITADS/ted_importer.py [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+                        manage_json_from_a(
+                            a, baseurl, headers, jsonfolder, outfilepath)
+
+                    file_counter += 1
+
+            pagenumber += 1
+        else:
+            the_end = True
+
+    print("Import from ted.com completed!)
```
Comment (Mte90) [inline MITADS/ted_importer.py:85]:
```suggestion
    print("Import from ted.com completed!")
```
ehm

**Comment by Mte90** [comment]
Just remove the trace stuff and this pr can be approved.

**Comment by dag7dev** [comment]
Leaving trace functions here as a reference:
```
'''
    write a string in a log_file
    log file will be saved in parsefolder
'''
def trace(string):
    parsefolder = "parsing"
    t = time.localtime()
    current_time = time.strftime("%H:%M:%S", t)

    fh = open(parsefolder + os.path.sep + "log.txt", "a+")

    fh.write(current_time + " : " + str(string) + "\n")
    fh.close()


'''
    clean log file
'''
def clean_log():
    parsefolder = "parsing"
    fh = open(parsefolder + os.path.sep + "ted_log.txt", "w+")
    fh.close()
```

**Pull request merged**